### DISCUSSION TITLE
读取数据库中的群历史数据

### DISCUSSION THREAD

**Discussion by hjlin0515** [Mrs4s/go-cqhttp#1986] [General]
请教一下，我看启动cqhttp之后，终端显示一直在接收数据，这部分是否有存数据库中，能否直接读取数据库拿这部分的数据？感谢！

**Comment by umena337** [comment]
请教一下，我看启动cqhttp之后，终端显示一直在接收数据，这部分是否有存数据库中，能否直接读取数据库拿这部分的数据？感谢！
